### ISSUE TITLE
export 'useInsertionEffect' (imported as 'React') was not found in 'react'

### ISSUE THREAD

**Issue by NickTheFreak97** [vingeraycn/ruuri#3]
Hi, I just installed the package using npm and got the following error straight away: 

```
in ./node_modules/ruuri/dist/index.es.js 1394:25-52
export 'useInsertionEffect' (imported as 'React') was not found in 'react' (possible exports: Children, Component, Fragment, Profiler, PureComponent, StrictMode, Suspense, __SECRET_INTERNALS_DO_NOT_USE_OR_YOU_WILL_BE_FIRED, cloneElement, createContext, createElement, createFactory, createRef, forwardRef, isValidElement, lazy, memo, useCallback, useContext, useDebugValue, useEffect, useImperativeHandle, useLayoutEffect, useMemo, useReducer, useRef, useState, version)
```

I'm compiling with React 17.0.2. I found a similar issue [here](https://namespaceit.com/blog/export-useinsertioneffect-imported-as-useinsertioneffect1-was-not-found-in-react) and it appears it might be linked to @emotion/react. I tried updating but didn't solve.

**Comment by vingeraycn** [comment]
Hi, nice to hear from you, you mentioned `@emotioncss` something that caught my attention, it just so happens that I had a compatibility issue with `@emotionocss` on another project some time ago, so I tried to remove the dependency on `@emotioncss` in the ruuri project, and now a prepatch version has been released ` v0.1.7-0`, if you have time, you can try to upgrade it.

Thanks a lot for your feedback again!

**Comment by vingeraycn** [comment]
Hi NickTheFreak, nice to hear from you, you mentioned @emotioncss something that caught my attention, it just so happens that I had a compatibility issue with @emotionocss on another project some time ago, so I tried to remove the dependency on @emotioncss in the ruuri project, and now a prepatch version has been released ` v0.1.7-1`, if you have time, you can try to upgrade it.

Thanks a lot for your feedback again!

**Comment by NickTheFreak97** [comment]
Hi, I'll try it later in the day and let you know how it went. Thanks for your time!

**Comment by NickTheFreak97** [comment]
So, I tested in in codesandbox, copying your React 17 demo and updating its package.json ruuri to 0.1.7-1 and [this](https://codesandbox.io/s/ruuri-on-react17-forked-dd2rqv?file=/src/App.js) is what I'm getting. Apparently the last item of the grid falls of its container and the drag and drop functionality is negatively affected from this. Hope this was useful.

**Comment by vingeraycn** [comment]
Thank you very much for your help, the problem you mentioned just now that you can't drag and drop properly is caused by the failure of CSS style injection, this problem has been fixed in v0.1.7-2 now, I forked the codesandbox you shared [here](https://codesandbox.io/s/ruuri-on-react17-forked-1eti39?file=/src/App.js) and upgraded ruuri to v0.1.7-2, the problem should be solved.

**Comment by NickTheFreak97** [comment]
Great, I checked it out and apparently it works correctly now. I think you can close this issue, and probably the other one too, since it appears to be related to this as well. Thank a lot, much appreciated!
